From a8a827ba8b2d5063b7a35799ab7658d1164371c5 Mon Sep 17 00:00:00 2001
From: Shawn Guo <shawn.guo@freescale.com>
Date: Tue, 23 Jul 2013 10:26:00 +0800
Subject: [PATCH 2/2] ENGR00240988: drm: vivante: remove reclaim_buffers
 callback

Since commit b0071ef (drm: kill reclaim_buffers callback),
reclaim_buffers callback is unavailable.  Remove it.

Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 drivers/gpu/drm/vivante/vivante_drv.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/vivante/vivante_drv.c b/drivers/gpu/drm/vivante/vivante_drv.c
index aa652a3..4caf46f 100644
--- a/drivers/gpu/drm/vivante/vivante_drv.c
+++ b/drivers/gpu/drm/vivante/vivante_drv.c
@@ -71,7 +71,6 @@ static const struct file_operations viv_driver_fops = {
 
 static struct drm_driver driver = {
 	.driver_features = DRIVER_USE_MTRR,
-	.reclaim_buffers = drm_core_reclaim_buffers,
 	.fops = &viv_driver_fops,
 	.name = DRIVER_NAME,
 	.desc = DRIVER_DESC,
-- 
1.8.5.2

